crates/turborepo-lib/src/run/summary/task.rs

```diff
@@ -241,6 +267,67 @@ impl From<SharedTaskSummary<TaskId<'static>>> for SharedTaskSummary<String> {
     }
 }
 
+impl From<TaskDefinition> for TaskSummaryTaskDefinition {
+    fn from(value: TaskDefinition) -> Self {
+        let TaskDefinition {
+            outputs:
+                TaskOutputs {
+                    inclusions,
+                    exclusions,
+                },
+            cache,
+            mut env,
+            pass_through_env,
+            dot_env,
+            topological_dependencies,
+            task_dependencies,
+            mut inputs,
+            output_mode,
+            persistent,
+        } = value;
+
+        let mut outputs = inclusions;
+        for exclusion in exclusions {
+            outputs.push(format!("!{exclusion}"));
+        }
+
+        let mut depends_on =
+            Vec::with_capacity(task_dependencies.len() + topological_dependencies.len());
+        for task_dependency in task_dependencies {
+            depends_on.push(task_dependency.to_string());
+        }
+        for topological_dependency in topological_dependencies {
+            depends_on.push(format!("^{topological_dependency}"));
+        }
+
+        // This should _not_ be sorted.
```

Comment:
I'm not sure why this comment existed in Go, but I'm copying it and it's behavior here.

Reply:
Dot env files are the one exception where we don't sort our inputs, because they override each other